### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by roland-5** [ouch-org/ouch#552] [General]
I'm asking more out of curiosity than need. There are interesting things in Pull Request like partial RAR and bzip3 support and I was wondering if you are planning a light release before or after adding these features?

**Comment by marcospb19** [comment]
Yes! I do plan on making a new release for when these land.

More specifically, about partial RAR, we need a new `unrar` release, or else it's not possible to make a new release to crates.io.

EDIT: looks like the new `unrar` release will happen soon, things are moving there.

**Reply by marcospb19** [reply]
#529 is merged, I should make a new release.

**Reply by marcospb19** [reply]
Another update: I tried making a new release, but our GitHub Action that help with that is broken, and I have no idea why, tried debugging and even asked Figsoda but he's clueless as well.

Might get delayed until I solve it.

**Reply by roland-5** [reply]
You did it, it worked! Many thanks for the new release. <3
